tst/testlibgap/basic.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Small program to test libgap linkability and basic working
+ */
+#include <stdio.h>
+#include <unistd.h>
+
+#include <compiled.h>
+#include <libgap-api.h>
+
+extern char ** environ;
+
+void test_eval(const char * cmd)
+{
+    Obj res, ires;
+    Int rc, i;
+
+    printf("gap> %s\n", cmd);
+    res = GAP_EvalString(cmd);
+    rc = GAP_LenPList(res);
+    for(i=1;i<=rc;i++) {
+        ires = GAP_ElmPList(res, i);
+        if( GAP_ElmPList(ires, 1) == True )
+            printf("%s\n", CSTR_STRING(GAP_ElmPList(ires, 5)));
+    }
+}
+
+int main(int argc, char **argv)
+{
+    printf("Initializing GAP...\n");
+    GAP_Initialize(argc, argv, environ, 0L, 0L);
+    printf("done\n");
```

Comment:
Optional: Maybe change to
```C
printf("# Initializing GAP...\n");
GAP_Initialize(argc, argv, environ, 0L, 0L);
printf("# done\n");
```
so that the output file becomes a valid .tst file?